ocamltest/tsl_ast.mli

```diff
@@ -35,6 +36,9 @@ type tsl_item =
 
 type tsl_block = tsl_item list
```

Comment:
In the context of the new syntax, the name `tsl_block` to mean "a toplevel script" is a bit confusing as it could equally mean a block as in C. I would remove this synonym and expand it away in the parser.

Reply:
This is from the old code and I haven't changed it. My plan is to remove the old language altogether in the follow-up PR (which will also change the log format and greatly simplify the AST code).

Reply:
This is from the old code and I haven't changed it. My plan is to remove it altogether in the follow-up PR that will remove the old syntax and the translation tool, refactor this AST code, and change the log format.